src/core/util.tsx

```
@@ -938,9 +947,23 @@ export function isValidPrivateKey(privateKey: string): boolean {
     return !!wallet;
   } catch (e) {
     return false;
+    return false;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)